### PR TITLE
feat(FactCommand): add factcommand

### PR DESCRIPTION
**Pull request by gautam-paranjape**
Added a little fact command that fetches random facts from an API. I made sure to run npm run lint, to make sure that the codestyle is the same.

### CODE REVIEW

File: src/commands/Fun Commands/FactCommand.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { RunFunction } from '../../interfaces/Command';
+import fetch from 'node-fetch';
+
+export const run: RunFunction = async (client, message) => {
+	let response: any = await fetch('https://no-api-key.com/api/v1/facts').then(
```
Comment (Milo123459) [inline src/commands/Fun Commands/FactCommand.ts:5]: use const over let

File: src/commands/Fun Commands/FactCommand.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+	);
+
+	await message.channel.send(
+		client.embed({ title: 'Random Fact!', description: response.fact }, message)
+	);
+};
+
+export const name: string = 'fact';
+export const category: string = 'fun';
+export const cooldown: number = 3000;
```
Comment (Milo123459) [inline src/commands/Fun Commands/FactCommand.ts:18]: exporting cooldown as 3 seconds is a bad idea as 3 is the default cooldown

Comment (gautam-paranjape) [reply src/commands/Fun Commands/FactCommand.ts:18]: ok should I do 4 then or something

File: src/commands/Fun Commands/FactCommand.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { RunFunction } from '../../interfaces/Command';
+import fetch from 'node-fetch';
+
+export const run: RunFunction = async (client, message) => {
+	let response: any = await fetch('https://no-api-key.com/api/v1/facts').then(
+		(res) => {
```
Comment (Milo123459) [inline src/commands/Fun Commands/FactCommand.ts:6]: dont do => { just do =>

File: src/commands/Fun Commands/FactCommand.ts (hunk 2 above)
Comment (Milo123459) [reply src/commands/Fun Commands/FactCommand.ts:18]: just remove the line

**Comment by Milo123459** [comment]
@gautam-paranjape Good first contribution over all, I recommend trying the yarn new command - it sets out everything for you.

File: src/commands/Fun Commands/FactCommand.ts (hunk 2 above)
Comment (gautam-paranjape) [reply src/commands/Fun Commands/FactCommand.ts:18]: ohh, so every command has a 3000MS cooldown, and you can also specify a different cooldown?

Comment (gautam-paranjape) [reply src/commands/Fun Commands/FactCommand.ts:18]: every command has a 3000MS cooldown by default*

Comment (Milo123459) [reply src/commands/Fun Commands/FactCommand.ts:18]: yes.

**Comment by gautam-paranjape** [comment]
Thanks, I will remember these changes next time I contribute, and yes I will use the yarn new command, didn't know that. I am pushing the changes to what you said now

**Review by Milo123459 — APPROVED** [review]

File: src/commands/Fun Commands/FactCommand.ts (hunk 2 above)
Comment (gautam-paranjape) [reply src/commands/Fun Commands/FactCommand.ts]: ok thanks

**Pull request merged**